hooks/use-transcription-indexer.ts

```
@@ -0,0 +1,273 @@
+      })
+    }
+  })
+
+  // Search transcriptions
+  const searchTranscriptions = useCallback(
+    (query: string, options: {
+      limit?: number
+      threshold?: number
+      sessionId?: string
+    } = {}) => {
+      return useQuery({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)